tests/integration/test_trading.py

```diff
@@ -294,3 +296,175 @@ def test_estimate_position(vega_service_with_market: VegaServiceNull):
         is_market_order=[False, False],
         collateral_available=1,
     )
+
+@pytest.mark.integration
+def test_liquidation_and_estimate_position(vega_service_with_market: VegaServiceNull):
+    vega = vega_service_with_market
+    WalletConfig = namedtuple("WalletConfig", ["name", "passphrase"])
+
+    MM_WALLET = WalletConfig("mm", "pin")
+    PARTY_A = WalletConfig("party_a", "party_a")
+    PARTY_B = WalletConfig("party_b", "party_b")
+    TERMINATE_WALLET = WalletConfig("TERMINATE", "TERMINATE")
+    LIQ_WALLET = WalletConfig("LIQ", "TRADER")
+    ASSET_NAME = "tDAI"
+    WALLETS = [MM_WALLET, PARTY_A, PARTY_B, TERMINATE_WALLET, LIQ_WALLET]
+
+    # vega: VegaServiceNull,
+    mint_amount = 10000
+    initial_volume = 10
+    initial_commitment = 1000
+    collateral_available = 1
+
+    vega.wait_for_total_catchup()
+    for wallet in WALLETS:
+        vega.create_key(wallet.name)
+
+    vega.wait_for_total_catchup()
+    vega.mint(
+        MM_WALLET.name,
+        asset="VOTE",
+        amount=1e4,
+    )
+    vega.forward("10s")
+
+    # Create asset
+    vega.create_asset(
+        MM_WALLET.name,
+        name=ASSET_NAME,
+        symbol=ASSET_NAME,
+        decimals=1,
+        max_faucet_amount=10 * mint_amount * 1e5,
+    )
+    vega.forward("10s")
+    vega.wait_for_total_catchup()
+
+    asset_id = vega.find_asset_id(symbol=ASSET_NAME)
+
+    for wallet in WALLETS:
+        vega.mint(
+            wallet.name,
+            asset=asset_id,
+            amount=mint_amount,
+        )
+    vega.forward("10s")
+    vega.create_simple_market(
+        market_name="CRYPTO:BTCDAI/DEC22",
+        proposal_key=MM_WALLET.name,
+        settlement_asset_id=asset_id,
+        termination_key=TERMINATE_WALLET.name,
+        market_decimals=1,
+    )
+
+    market_id = vega.all_markets()[0].id
```

Comment:
The current fixture `vega_service_with_market` also creates a market so this method may not return the market id you are after. Try switching the fixture so there is only one market on then network.

For extra safety use the `find_market_id` method instead and pass the market name.